tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
         >>> data = [img1, img2]
```

Comment:
`Examples:` instead of usage.

Reply:
Please see this
https://tia-toolbox.readthedocs.io/en/latest/_modules/tiatoolbox/models/classification/patch_predictor.html#CNNPatchPredictor

Reply:
Did you pull changes from develop?

Reply:
This is stale comment so i will just resolve this. This should have been addressed in improve model doc PR